src/components/RatingsChart/RatingsChart.tsx

```diff
@@ -802,23 +932,26 @@ export class RatingsChart extends React.PureComponent<RatingsChartProperties, an
     }}}
 
     render() {{{
+        this.computeWinLossNumbers();
+        if (!this.state.loading && this.show_pie) {
+            this.plotWinLossPie();
+        }
         return (
             <div ref={(e) => this.container = e} className="RatingsChart">
                 {this.state.loading
                     ? <div className='loading'>{_("Loading")}</div>
                     : this.state.nodata
                         ? <div className='nodata'>{_("No rated games played yet")}</div>
-                        : <PersistentElement elt={this.chart_div}/>
+                        : <div className='ratings-graph'>
+                            <PersistentElement elt={this.chart_div}/>
+                        </div>
                 }
-                {this.renderWinLossNumbers()}
+                {this.show_pie ? "" : this.renderWinLossNumbersAsText()}
```

Comment:
Should be `null` instead of `""` as putting in a `""` will actually create a text node